ethereum/contracts/test/VerifierWeb3.sol

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.6.2;
+
+contract VerifierWeb3 {
+
+    address public operator;
+
+    constructor(address _operator) public {
+        operator = _operator;
+    }
+
+    function recover(bytes32 _message, bytes memory _signature)
+        public
+        view
+        returns (bool)
+    {
+        address signer = verify(ethMessageHash(_message), _signature);
+        return operator == signer;
+    }
+
+  /**
+     * @dev Verify checks if the signer's address matches the operator's address
+     */
+    function verify(bytes32 h, bytes memory signature)
```

Comment:
Same as above regarding function names and docs

Reply:
https://github.com/Snowfork/polkadot-ethereum/commit/f2fe2d84c111628cbc300984e643275cbcc0c8dd